common/lib/dependabot/shared_helpers.rb

```diff
@@ -484,6 +484,95 @@ def self.helper_subprocess_bash_command(command:, stdin_data:, env:)
       env_keys = env ? env.compact.map { |k, v| "#{k}=#{v}" }.join(" ") + " " : ""
       "$ cd #{Dir.pwd} && echo \"#{escaped_stdin_data}\" | #{env_keys}#{command}"
     end
+
```

Comment:
Let's move these new methods into the PackageManager class, the SharedHelpers are more for executing shell commands and doing things with the file system.